### PR TITLE
get rid of global, use css modules

### PR DESCRIPTION
**Pull request by jgfidelis**
https://prodcss--phmex.myvtex.com/super_suprema/p
https://prodcss--delivery.myvtex.com/pizza-pepperoni/p

### CODE REVIEW

File: react/styles.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.multipleChoiceContainer {}
+
+.toggleChoiceContainer {}
+
+.multipleChoiceActions {}
+
+.multipleChoiceActions button {
+  background: transparent;
+  width: 1.75rem !important;
```
Comment (klzns) [inline react/styles.css]: check if this is really needed

File: react/styles.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+.singleChoiceContainer {}
+
+.multipleChoiceContainer {}
+
+.toggleChoiceContainer {}
+
+.multipleChoiceActions {}
+
+.multipleChoiceActions button {
+  background: transparent;
```
Comment (klzns) [inline react/styles.css]: can use tachyons'`bg-transparent` instead

File: react/components/Buttons/MovingBottomButton.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ class MovingBottomButton extends PureComponent {
     return (
       <Fragment>
         <div
-          className={`vtex-product-customizer__actions w-100`}
+          className="w-100"
```
Comment (brunojdo) [inline react/components/Buttons/MovingBottomButton.js]: Why don't add a `styles.actions` here?

File: react/components/Buttons/MovingBottomButton.js [hunk 4]
```diff
@@ -58,7 +58,7 @@ class MovingBottomButton extends PureComponent {
           />
         </div>
         <div
-          className={`vtex-product-customizer__actions z-2 fixed bg-white bottom-0 w-40-ns ph1-ns w-100 bt b--light-gray`}
+          className="z-2 fixed bg-white bottom-0 w-40-ns ph1-ns w-100 bt b--light-gray"
```
Comment (brunojdo) [inline react/components/Buttons/MovingBottomButton.js]: same

File: react/components/Buttons/MovingBottomButton.js (hunk 3 above)
Comment (jgfidelis) [reply react/components/Buttons/MovingBottomButton.js]: Tinha esquecido de add e commit um arquivo com o `addToCartButton` class. Ele é o mesmo que adicionar aí, mas num lugar único.

File: react/components/Variation/Items/ToggledChoice.js [hunk 5]
```diff
@@ -2,23 +2,25 @@ import React, { memo } from 'react'
...
+import ItemDescription from './ItemDescription'
+
+import styles from '../../../styles.css'
 
 const ToggledChoice = ({ imageUrl, name, selected, onChange, disabled }) => (
-  // eslint-disable-next-line jsx-a11y/label-has-associated-control
-  <label
-    className={`vtex-product-customizer__toggled-choice ${
+  <div
+    className={`${styles.toggleChoiceContainer} ${
```
Comment (matheusps) [inline react/components/Variation/Items/ToggledChoice.js]: `classNames` maybe ? 🤔

File: react/components/Variation/Items/MultipleChoice.js [hunk 6]
```diff
@@ -36,18 +32,25 @@ const MultipleChoice = ({
...
         }}
       />
-    </div>
+    </Fragment>
   )
 
   return (
     <div
-      className={`vtex-product-customizer__multiple-choice ${isSelected &&
+      className={`${styles.multipleChoiceContainer} ${isSelected &&
```
Comment (matheusps) [inline react/components/Variation/Items/MultipleChoice.js]: `classNames` as well

File: react/components/Variation/Items/ToggledChoice.js [hunk 7]
```diff
@@ -2,23 +2,25 @@ import React, { memo } from 'react'
...
-      <div className={`${disabled ? 'o-70' : ''}`}>
+    <div className="relative flex justify-between items-center">
+      <ItemDescription 
+          name={name}
+          imageUrl={imageUrl}
+          containerClass={styles.itemDescriptionContainerToggle}
+          nameClass={styles.itemDescriptionNameToggle}
+          descriptionClass={styles.itemDescriptionToggle}
+        />
+      <div className={`${styles.toggleChoiceActions} ${disabled ? 'o-70' : ''}`}>
```
Comment (matheusps) [inline react/components/Variation/Items/ToggledChoice.js]: Here too

**Pull request closed without merging**